lib/contestant.rb

```diff
@@ -0,0 +1,26 @@
+class Contestant
+  attr_reader :first_name, :last_name, :age, :state_of_residence, :spending_money, :game_interests
+
+  def initialize(attribute)
+    @first_name         = attribute[:first_name]
+    @last_name          = attribute[:last_name]
+    @age                = attribute[:age]
+    @state_of_residence = attribute[:state_of_residence]
+    @spending_money     = attribute[:spending_money]
+    @out_of_state       = false
```

Comment:
A contestant will not _always_ be out of state - we should use the `@state_of_residence` to determine this with a method:

```ruby
def out_of_state?
  if @state_of_residence == "CO"
    false
  else
    true
  end
end
```